src/pages/workspaces/migration/WorkspaceItem.ts

```diff
@@ -0,0 +1,115 @@
+import { div, h, span } from 'react-hyperscript-helpers';
+import { icon } from 'src/components/icons';
+import { InfoBox } from 'src/components/PopupTrigger';
+import colors from 'src/libs/colors';
+import * as Utils from 'src/libs/utils';
+import { WorkspaceMigrationInfo } from 'src/pages/workspaces/migration/migration-utils';
+
+export const WorkspaceItem = (workspaceInfo: WorkspaceMigrationInfo) => {
+  const renderMigrationState = () => {
+    const getTransferProgress = (transferType, processed, total) => {
+      if (total === 0) {
+        return `${transferType} Bucket Transfer`;
```

Comment:
If we have no bytes to transfer, I tried to clarify that we are transferring the bucket itself.